src/BenchmarkDotNet/BenchmarkDotNet.csproj

```
@@ -2,8 +2,9 @@
   <Import Project="..\..\build\common.props" />
   <PropertyGroup>
     <AssemblyTitle>BenchmarkDotNet</AssemblyTitle>
-    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net6.0</TargetFrameworks>
```

In a few weeks from now (Dec 13 2022), .NET Core 3.1 will be out of support and .NET 6 will be the oldest supported .NET Standard 2.1 runtime. Since we already target `net6.0` I don't think it's worth adding `netstandard2.1` to the list.

```suggestion
    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
```

---

There's also Xamarin and Unity which are still supported.

---

We don't support Unity (it would require a lot of extra work) and Xamarin works just fine with .NET Standard 2.0.

---

2.1 would remove a couple more dependencies and avoid reflection in `IsAot`.

In my projects I usually target .NET Standard 2.0 for compatibility, .NET 6 for new features and .NET Standard 2.1 for everything in between, but I understand you not wanting this (the most obvious reason being that the 2.1 binaries won't be tested).

---

> We don't support Unity (it would require a lot of extra work) and Xamarin works just fine with .NET Standard 2.0.

#1860 is an open issue for that.